.github/workflows/build_and_publish_ecr.yml

```diff
@@ -0,0 +1,115 @@
+name: Build & Publish Docker Image to AWS ECR
+
+on:
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: "Log level"
+        required: true
+        default: "warning"
+
+env:
+  CARGO_TERM_COLOR: always
+  DOCKER_BUILDKIT: 1
+  RUSTFLAGS: -D warnings
+  REGISTRY: 756756437019.dkr.ecr.us-east-1.amazonaws.com/fuel-core
```

Comment:
Can we put this in a real env var in github rather than in a yaml file?

Reply:
Let's use Github Secrets, that would be the best practice